app/src/main/java/com/hedvig/app/feature/settings/MarketManager.kt

```diff
@@ -21,6 +23,7 @@ class MarketManagerImpl(
         Market.SE,
         Market.NO,
         Market.DK,
+        if (isFeatureEnabled(Feature.FRANCE_MARKET, market)) Market.FR else null
```

Comment:
Would it perhaps be simpler to simply make a `isDebug`-check here, and keep the rest of the architecture as-was? 🤔

Reply:
Yes that makes more sense. Will revert to that.